plans/prepare.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+RHEL_RELEASE=$(rpm --eval %{?rhel})
```

Comment:
Suggestion: singlequotes: `--eval '%{?rhel}'`

Reason: curly braces and question marks are special shell characters. It looks like bash handles this particular case, but quotes will make it 100% clear to a future maintainer.

Reply:
done